.java-version

```diff
@@ -1 +1 @@
-temurin-17.0.16+8
+temurin-21.0.8+9.0.LTS
```

Comment:
perhaps this should stay at 17? I'm thinking language level and api compatibility for those who wants to contribute thinking they can use 21 features but the PR build fails building with 17?